pages/profile/collections/index.js

```diff
@@ -1,40 +1,55 @@
-import IconButton from "@/components/Styled/IconButton";
 import { useRouter } from "next/router";
 import { useState } from "react";
 import updateUserinDb from "@/helpers/updateUserInDb";
-import Header from "@/components/Styled/Header";
+
 import styled from "styled-components";
-import StyledInput from "@/components/Styled/StyledInput";
-import Plus from "/public/icons/svg/plus.svg";
-import AddButton from "@/components/Styled/AddButton";
+import IconButton from "@/components/Styled/IconButton";
+import Plus from "@/public/icons/svg/plus.svg";
 import CollectionCard from "@/components/CollectionCard";
 import StyledH2 from "@/components/Styled/StyledH2";
+import NewCollection from "../../../components/Forms/NewCollection";
+import MenuContainer from "@/components/MenuContainer";
+import Pen from "/public/icons/svg/pen-square_10435869.svg";
+import Trash from "/public/icons/svg/trash-xmark_10741775.svg";
+import XSmall from "@/public/icons/XSmall.svg";
 
 export default function Collections({ user, mutateUser }) {
   const [addCollection, setAddCollection] = useState(false);
+  const [menuVisible, setMenuVisible] = useState(false);
```

Comment:
since these states are booleans, they should start with "is". This makes it clear from reading just the name of the states that their data type is a boolean.